File: h2o-docs/src/booklets/v2_2015/source/DeepLearning_Vignette_code_examples/deeplearning_importfile_example.py
import h2o

# Start H2O cluster with all available cores (default)
h2o.init() 

train = h2o.import_file("https://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/mnist/train.csv.gz")
test = h2o.import_file("https://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/mnist/test.csv.gz")

# Get a brief summary of the data
train.describe()
test.describe()

File: h2o-docs/src/booklets/source/DeepLearning_Vignette_code_examples/deeplearning_importfile_example.R
library(h2o)

# Sets number of threads to number of available cores
h2o.init(nthreads = -1)

train_file <- "https://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/mnist/train.csv.gz"
test_file <- "https://h2o-public-test-data.s3.amazonaws.com/bigdata/laptop/mnist/test.csv.gz"

train <- h2o.importFile(train_file)
test <- h2o.importFile(test_file)

# Get a brief summary of the data
summary(train)
summary(test)
